packages/grafana-runtime/src/utils/DataSourceWithBackend.ts

```diff
@@ -221,23 +261,43 @@ class DataSourceWithBackend<
   /**
    * Make a GET request to the datasource resource path
    */
-  async getResource(
+  async getResource<T = any>(
```

Comment:
I am a bit late to make this comment but can't we use `unknown` instead of `any`?